### PR TITLE
Minor adjustments to Windows build instructions

### PR DESCRIPTION
**Pull request by troughton**
This pull request contains a few minor changes to the Windows/MSVC build instructions to make them easier to follow.

I removed the time estimates from the instructions, both for clarity and because they're very system-dependent.

### CODE REVIEW

**Review by dabrahams — CHANGES_REQUESTED** [review]
Please put some indication of the time back.  One point of having that information is to give people a sense of what steps they should choose to skip based on their needs.  In particular, almost nobody ever needs to build compiler-RT

**Comment by troughton** [comment]
I can add some indication back, but I think how it's currently written doesn't make that clear; it's written currently as a series of steps (build CMark, then LLVM/Clang/compiler-RT **in one step**, then Swift), that presumably you'd all need to carry out to build Swift (which I'd imagine is the aim).

I can put in a note about compiler-RT being optional, which would definitely be useful information – I didn't realise that myself. As for timeframes – I'd argue that knowing which steps are necessary is more important, since you very quickly get a time indication once it starts building. I also haven't personally found the times mentioned to be at all representative, so perhaps "could take a few hours" might be better than "could take up to five hours"?

I'll defer to your judgement if you'd rather leave the time estimates as is; otherwise, I can have a try at rewriting to hopefully make things clearer.

**Comment by troughton** [comment]
I've added the time estimates back in and made a few other minor changes. I tried to soften the timing and give more reasons for the timeframes.

I think what I mainly took issue with in the way it was originally written was it seemed too definite; I read "could take up to 5 hours" and processed it as "will take between four to five hours, and no longer than five and a half". My rewritten version is much more verbose, however; I hope it's clearer, but if not, I can revert it back and keep the other changes.

File: docs/Windows.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ set swift_source_dir=path-to-directory-containing-all-cloned-repositories
 ```
 
 ### 5. Build CMark
-- This must be done from within a developer command prompt, and could take up to 10 minutes.
+- This must be done from within a developer command prompt. CMark is a fairly small project, and it should only take a few minutes to build.
```
Comment (CodaFi) [inline docs/Windows.md]: > project, and

Nit: Conjunction XOR comma

Comment (troughton) [inline docs/Windows.md]: Since we're nitpicking: I was actually curious and looked this up. Apparently, my change as written is correct; you need a comma before a conjunction when it's connecting two independent clauses. As such, the old version,
> prompt, and could
was incorrect, since that's a dependent clause, but my modification is actually correct.

However, the question is more whether it should be two independent clauses. I think it's probably clearer/more natural as a dependent clause, so I've changed it to match (and now the comma is gone.) I also went through and addressed the same concern in other places in the same file.

**Comment by CodaFi** [comment]
LGTM

@swift-ci please smoke test and merge

**Comment by CodaFi** [comment]
@swift-ci please smoke test and merge

**Comment by CodaFi** [comment]
libDispatch failed to build.

@swift-ci please smoke test linux platform

**Comment by CodaFi** [comment]
@swift-ci please smoke test linux platform

**Comment by troughton** [comment]
Checking in here – I believe we were blocked from merging earlier because of other issues, but it would be good to get these changes in (assuming nothing else needs to be modified).

**Comment by RUSshy** [comment]
When will you guys release swift for Windows ? i want make crosspaltform games with SDL, please do it!!

**Comment by troughton** [comment]
@RUSshy I can't speak for the Swift team, but I'd imagine a Windows port is very low priority considering all of the other important tasks (such as ABI stability). As such, all effort towards it at this stage has come from the community; I've done some very minor patches, and others have done far more substantial work towards it (see e.g. [Swift for Windows](https://swiftforwindows.github.io)). 

Unfortunately, this currently seems fairly disparate; different components and fixes needed are spread across multiple repositories, with only a subset consolidated into the official repo via pull requests.

Even after the main Swift compiler (which currently builds) and the standard library (which currently doesn't) are working on Windows, there's still a lot of effort needed into supporting libraries such as Dispatch, Foundation, the Package Manager, and LLDB to make Windows a first-class development platform for Swift.

In short, if you want to see a recent build of Swift on Windows soon, the best way to achieve that is to find some way to contribute yourself. If that seems too daunting (I personally only have a surface-level understanding of what's needed) then I'm afraid it's likely to be a reasonable wait.

**Comment by CodaFi** [comment]
@troughton Sorry to have this fall by the wayside.  I believe you've addressed both Dave and my comments, so

⛵️
